UK_Reports/reports/account_invoice_report.xml

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+	<data>
+		<template id="proforma_invoice_body">
+			<t t-set="doc_header_string" t-value="'Sales PRO-FORMA'"/>
+			<t t-call="UK_Reports.template_document_title"/>
+			<div class="row">
+				<!-- Space the address to the right slightly -->
+				<div class="col-xs-1"/>
+				<div class="col-xs-5">
```

Comment:
```suggestion
				<div class="col-xs-5 col-offset-1">
```

Reply:
Applied suggestion (after trialling the 'proper' way to do it) on all instances where offset was done manually with an empty div. see e2a44f1